Bottom: c69e4ab8b25196c012f0929c96fa30c6eb45019a
Top:    424751b3a7001322d82d71230a77cf8df259a942
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-08 11:43:06 +0100

Refresh of render-polygons-a-contour-at-a

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 1d07dec..453c7f9 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -599,7 +599,6 @@ hidgl_fill_pcb_polygon (PolygonType *poly, const BoxType *clip_box, double scale
   PLINE *contour;
   struct do_hole_info info;
 
-
   global_scale = scale;
 
   if (poly->Clipped == NULL)
